packages/headless/src/app/commerce-ssr-engine/types/common.ts

```diff
@@ -141,6 +141,21 @@ export interface ControllerDefinitionOption {
   search?: boolean;
 }
 
+interface UniversalController {
```

Comment:
I wonder if we should also have an "undefined" or "standalone" SolutionType for cases where all you have is a standalone search box and the rest of the page isn't Coveo-driven...